### PR TITLE
Add Examples folder

### PR DESCRIPTION
**Pull request by icpenguins**

### CODE REVIEW

File: examples/readme.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Fluentd -> New Relic Examples
+
+## Creating a Docker Image
+
+Using [Fluentd](https://www.fluentd.org/) as an image makes it simple to deploy a quick input logging solution for functions such as [Syslogs](https://docs.fluentd.org/input/syslog), [HTTP](https://docs.fluentd.org/input/http), custom [UDP](https://docs.fluentd.org/input/udp) and [TCP](https://docs.fluentd.org/input/tcp) use cases, [SNMP](https://github.com/iij/fluent-plugin-snmp), along with many other functions. The [Fluentd](https://www.fluentd.org/) team has put together a great [set of documents](https://docs.fluentd.org/container-deployment) to help you get their basic configuration setup. After that, you will want to get your logs flowing into [New Relic Logs](https://docs.newrelic.com/docs/logs/new-relic-logs/get-started/introduction-new-relic-logs) to create alerts and monitor your systems.
+
+If you are able to use the Fluentd image directly, it is really simple build on that image and add the New Relic Fluetd Output Plugin. The below set of steps assumes you have some basic understanding of building a Docker image.
```
Comment (bmcfeely) [inline examples/readme.md]: Fluetd Output Plugin -> Flue**n**td Output Plugin

File: examples/readme.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+# Make the directory
+sudo mkdir /etc/fluentd
+
+# Edit the file
+sudo nano /etc/fluentd/fluentd.conf
+```
+
+#### 2. Add the following to the `fluentd.conf and save the file
+
+```xml
```
Comment (bmcfeely) [inline examples/readme.md]: Might just reference the fluentd-syslog.conf file, if you had to change something in the config, you'd have to change it here and in that .conf file.

File: examples/readme.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+If you are able to use the Fluentd image directly, it is really simple build on that image and add the New Relic Fluetd Output Plugin. The below set of steps assumes you have some basic understanding of building a Docker image.
+
+### Docker Image: Steps
+
+#### 1. Create a `Dockerfile`
+
+It doesn't take much to get the New Relic Output Plugin into a docker image. Here is a good example from Docker on how best to create an image, [LINK](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/).
+
+```yaml
```
Comment (bmcfeely) [inline examples/readme.md]: Might just reference the Dockerfile, if you had to change something in the file, you'd have to change it here and in the Dockerfile.

File: examples/readme.md (hunk 1 above)
Comment (icpenguins) [reply examples/readme.md]: Thanks!

File: examples/readme.md (hunk 2 above)
Comment (icpenguins) [reply examples/readme.md]: Moved out of readme...

File: examples/readme.md (hunk 3 above)
Comment (icpenguins) [reply examples/readme.md]: Moved out of readme...

**Comment by icpenguins** [comment]
Fixed spelling, moved source out of readme and provided reference. Created a subdirectory to support more examples.

**Pull request merged**